### ISSUE TITLE
log spam

### ISSUE THREAD

**Issue by SettingDust** [AHilyard/Iceberg#5]
- Mod version: 1.0.34
 - Modloader: forge
 - Minecraft version: 1.16.5

https://github.com/AHilyard/Iceberg/blob/1.16.5/src/main/java/com/anthonyhilyard/iceberg/util/ItemColor.java#L71

```
[21:01:30] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:30] [Render thread/INFO] (iceberg) item: 下界金矿石
[21:01:31] [Render thread/INFO] (iceberg) item: 绯红木板
[21:01:31] [Render thread/INFO] (iceberg) item: 樱桃木板
[21:01:31] [Render thread/INFO] (iceberg) item: 梾木木板
[21:01:31] [Render thread/INFO] (iceberg) item: 丝兰木板
[21:01:31] [Render thread/INFO] (iceberg) item: 丝兰木板
[21:01:31] [Render thread/INFO] (iceberg) item: 丝兰木板
[21:01:32] [Render thread/INFO] (iceberg) item: 酸枝木板
[21:01:32] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:33] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:34] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:34] [Render thread/INFO] (iceberg) item: 丝兰木板
[21:01:34] [Render thread/INFO] (iceberg) item: 金矿石
[21:01:34] [Render thread/INFO] (iceberg) item: 红沙
[21:01:34] [Render thread/INFO] (iceberg) item: 绯红木板
[21:01:34] [Render thread/INFO] (iceberg) item: 诡异木板
[21:01:34] [Render thread/INFO] (iceberg) item: 梾木木板
[21:01:34] [Render thread/INFO] (iceberg) item: 铁矿石
[21:01:35] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:35] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
[21:01:35] [Render thread/INFO] (iceberg) item: §a掠夺者 护符碎片
```

**Comment by AHilyard** [comment]
Thank you for pointing this out, it will be fixed in the next update.

Related: AHilyard/LegendaryTooltips#2

**Comment by AHilyard** [comment]
This has been fixed with the latest release (1.0.38), [here](https://www.curseforge.com/minecraft/mc-mods/iceberg/files/3609064).
